### ISSUE TITLE
Switch to full quarto website

### ISSUE THREAD

**Issue by adammwilson** [eco-emma/emma_report#21]
I think it's time to drop the google sites wrapper around the quarto website.  Can we switch to a full quarto website (instead of separate HTML pages)? Then we could have dropdown menus with pages for model, each park, etc.

**Comment by bmaitner** [comment]
Yeah, that would look better.  I'll get on this once I finish the content/plot updates

**Comment by adammwilson** [comment]
great, thanks.
